src/UserAgreement/ReduxCertPopup.js

```diff
@@ -23,9 +23,10 @@ export const submitForm = (data, questionList, quiz) => (dispatch) => fetchWithC
 })
   .then(
     ({ status }) => {
+      const cleanBasename = basename.replace(/^\/+/g, '').replace(/(dev.html$)/, '');
```

Comment:
Should `cleanBasename` ever change? 
Sometimes I see 

`const cleanBasename = basename.replace(/(dev.html$)/, '');`

And sometimes I see 

`const cleanBasename = basename.replace(/^\/+/g, '').replace(/(dev.html$)/, '');`

I think it would be cleaner to always use the second one and export it from `localconf`

Reply:
Yeah I agree this is confusing 😕 

So the differences is `basename.replace(/^\/+/g, '').replace(/(dev.html$)/, '')` actually takes out the leading `/` in the basename, if any. This is being used when the base name will be appending to somewhere that will have a trailing `/` later, for example, the `hostname` in portal. While the first one keeps the leading `/` in there.

we maybe should do an overhaul over all those names and paths in portal so they are more consistent, but I think we can create anotehr ticket for that task

Reply:
Or maybe we can always export the clean base name without the leading `/` from localconf, and add a leading `/` to it when needed?

Reply:
That makes sense. Exporting the clean base name without a `/` sounds slightly cleaner, but it may not be worth it so I think this is fine